ow_lander/launch/spawn.launch

```diff
@@ -102,12 +102,6 @@
   
   <!-- == launch the action servers ============== -->
   <arg name="node_start_delay" default="10.0" />  
-  <node pkg="ow_lander" name="Unstow" type="unstow_action_server.py" launch-prefix="bash -c 'sleep $(arg node_start_delay); $0 $@' " />
-  <node pkg="ow_lander" name="Stow" type="stow_action_server.py" launch-prefix="bash -c 'sleep $(arg node_start_delay); $0 $@' " />
-  <node pkg="ow_lander" name="Deliver" type="deliver_action_server.py" launch-prefix="bash -c 'sleep $(arg node_start_delay); $0 $@' " />
-  <node pkg="ow_lander" name="DigLinear" type="dig_linear_action_server.py" launch-prefix="bash -c 'sleep $(arg node_start_delay); $0 $@' " />
-  <node pkg="ow_lander" name="DigCircular" type="dig_circular_action_server.py" launch-prefix="bash -c 'sleep $(arg node_start_delay); $0 $@' " />
-  <node pkg="ow_lander" name="Grind" type="grind_action_server.py" launch-prefix="bash -c 'sleep $(arg node_start_delay); $0 $@' " />
-  <node pkg="ow_lander" name="GuardedMove" type="guarded_move_action_server.py" launch-prefix="bash -c 'sleep $(arg node_start_delay); $0 $@' " />
-  <node pkg="ow_lander" name="AntennaPanTiltAction" type="antenna_pan_tilt_action_server.py" launch-prefix="bash -c 'sleep $(arg node_start_delay); $0 $@' " />
+  <node pkg="ow_lander" name="AllArmActionServers" type="all_arm_action_servers.py" launch-prefix="bash -c 'sleep $(arg node_start_delay); $0 $@' " />

```

Comment:
I would drop the "all_" from the `all_arm_action_servers.py`
```suggestion
  <node pkg="ow_lander" name="ArmActionServers" type="arm_action_servers.py" launch-prefix="bash -c 'sleep $(arg node_start_delay); $0 $@' " />
```

Reply:
Done!